shared-libs/search/src/generate-search-requests.js

```diff
@@ -1,4 +1,6 @@
-const _ = require('underscore');
+const _ = require('lodash/core');
+_.partial = require('lodash/partial');
+_.partial.placeholder = _;
```

Comment:
What? `lodash/partial` needs a reference to `lodash/core`?

Reply:
Well, yes and no. 
the _.partial needs a `placeholder` entity that it checks with an === when creating the partial. 
We can give it any placeholder we want, as long as we use it when declaring calling _.partial. 

So we can do:
```
const partial = require('lodash/partial');
partial.placeholder = '___PLACEHOLDER__';
....... < code > ......
const myFn = (p1, p2, p3, p4) => p1 + p2 + p3 + p4;
const partialFn = partial(myFn, 2, 3, '___PLACEHOLDER__', 5);
partialFn(10) //20
```

I  just assigned it the _ object as a placeholder so I wouldn't touch the existing code and replace.

Reply:
Because `partial` is not part of `lodash/core`, we're initializing it independently and it doesn't have a placeholder assigned. 
There is a magic string that lodash uses internally, but that might change so it's not safe to use. 
If we intend to use the partial with the placeholder, we need to define it ourselves.

Reply:
For a complete reference, this is how this assignment is done in `lodash.js`:
```
// Assign default placeholders.
    arrayEach(['bind', 'bindKey', 'curry', 'curryRight', 'partial', 'partialRight'], function(methodName) {
      lodash[methodName].placeholder = lodash;
    });
```